Code/Editor/editor_lib_camera_test_files.cmake

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+set(FILES
+    Lib/Tests/test_EditorCamera.cpp
```

Comment:
you could put this file and `test_EditorCamera.cpp` in `Lib/Tests/Camera/` so it is easier to have it isolated of other targets

Reply:
Good shout, now moved :+1: